rs/sns/governance/canister/canister.rs

```diff
@@ -154,7 +145,9 @@ impl Environment for CanisterEnv {
             /* message: */ String,
         ),
     > {
-        call_bytes_with_cleanup(canister_id, method_name, &arg, Funds::zero()).await
+        ic_cdk::api::call::call_raw(canister_id.get().0, method_name, &arg, 0)
```

Comment:
Could you add a comment explaining why we're not using just `ic_cdk::call` here?

Reply:
done